make_wheels.py

```diff
@@ -128,25 +189,49 @@ def write_ziglang_wheel(out_dir, *, version, platform, archive):
     import subprocess; sys.exit(subprocess.call(argv))
 '''.encode('ascii')
 
+    # 1. Check for missing required licenses paths
+    missing_licenses = set(required_license_paths) - found_license_files
+    if missing_licenses:
+        print(f"\033[93mWarning: the following required license files were not found in the Zig archive: {', '.join(sorted(missing_licenses))} "
+              "\nThis may indicate a change in Zig's license file structure or an error in the listing of license files and/or paths.\033[0m")
+
+    # 2. Check for potentially missing license files
+    extra_licenses = potential_extra_licenses - set(required_license_paths)
+    if extra_licenses:
+        print(f"\033[93mWarning: found additional potential license files in the Zig archive but not included in the metadata: {', '.join(sorted(extra_licenses))} "
+              "\nPlease consider adding these to the license paths if they should be included.\033[0m")
```

Comment:
I feel like these are best done as hard errors, since this type of tool is often run rather mindlessly. If we ever need to bypass them we can add a flag.

Reply:
Yes, good shout. I agree that license compliance should be more strict in general.

Reply:
Addressed in 7c0d654bc2269da450b40007f971b32d4567086f.